geometry_msgs/Pose2D current_mobile_state
geometry_msgs/Pose2D target_mobile_pose
mobile_manipulator_controller/Obstacle2D[] Obstacles2D
---
mobile_manipulator_controller/MobileTrajectory mobile_trajectory
